packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -72,7 +74,11 @@ class CallContext {
   private state: CallAdapterState;
   private callId: string | undefined;
 
-  constructor(clientState: CallClientState, isTeamsCall: boolean) {
+  constructor(
+    clientState: CallClientState,
+    isTeamsCall: boolean,
+    /* @conditional-compile-remove(rooms) */ roleHint?: undefined | /* @conditional-compile-remove(rooms) */ Role
```

Comment:
Isn't this enough? `/* @conditional-compile-remove(rooms) */ roleHint?: Role

Reply:
No it was not enough. Functions with beta type arguments still need the beta type reference in stable for some reason. So I had to hack it with an `undefined` union to get cond-comp to work. But it is a moot point, after moving roleHint to an object type argument.